scripts/deploy.py

```diff
@@ -91,6 +95,13 @@ def get_contract_txn(contract_type: str, global_schema, **format_data):
     prompt="Fee bps",
     help="Fee in basis points taken from the outcome of each swap.",
 )
+@click.option(
+    "--version",
+    type=int,
+    default=1,
```

Comment:
Maybe `default=None`, which will pick the current version (the one without `_v_x` suffix).
That way, we won't have to update this place with each new version.